.github/scripts/bot-inactivity-unassign.sh

```
@@ -43,183 +40,79 @@ parse_ts() {
-    else
-      # Safety valve: if assignment event is missing, skip checking to prevent false positives
-      echo "    [WARN] Could not find 'assigned' event in timeline. Skipping inactivity check for safety."
+    if [[ -z "$ASSIGN_EVENT_JSON" || "$ASSIGN_EVENT_JSON" == "null" ]]; then
       continue
     fi
 
-    echo "    [INFO] Assignment source: $ASSIGN_SOURCE"
-    echo "    [INFO] Assigned at:      ${ASSIGNED_AT:-(unknown)} (~${ASSIGNED_AGE_DAYS} days ago)"
+    ASSIGNED_AT=$(echo "$ASSIGN_EVENT_JSON" | jq -r '.created_at')
+    ASSIGNED_TS=$(parse_ts "$ASSIGNED_AT")
+    ASSIGNED_AGE_DAYS=$(( (NOW_TS - ASSIGNED_TS) / 86400 ))
```

Shellcheck (0.11.0)

[warning] ASSIGNED_AGE_DAYS appears unused. Verify use (or export if used externally).

(SC2034)